### ISSUE TITLE
Specify Files Location

### ISSUE THREAD

**Issue by adrian-chang** [jackfranklin/pulldown#15]
Hey thanks for the update.

This is getting pretty close.

The only thing I'd say is missing is specifying where a file is located after a pulldown happens.

IE I'd want to be able to put jquery into a ./vendor folder etc. instead of dumping it into the local folder. If this options already exists, I'm not sure I see it. It's not really possible to do this as far as I can see unless you make a per folder pulldown.json.

**Comment by jackfranklin** [comment]
So I'm thinking something like:

```
pulldown jquery vendor/
pulldown backboneapp vendor/
```

Which would send all the files into the `vendor/` directory.

I also would like to support `pulldown jquery:foo.js` to download jQuery to `foo.js`. That would only work when the thing you're downloading is just one URL, though.

**Comment by tgvashworth** [comment]
In theory we should support pulling down multiple packages at the same time:

```
pulldown angular backbone
```

So what about an option (`--target` or `--out`?) that specifies the target directory?

```
pulldown jquery -t vendor/
pulldown backboneapp -o vendor/
```

This would then support the `name:file` syntax too.

```
pulldown jquery:j.js --target vendor/
```

**Comment by jackfranklin** [comment]
:+1: I like that.

**Comment by jackfranklin** [comment]
This landed in `0.2.3`.
